appium/README.md

```diff
@@ -9,10 +9,14 @@
 5. restart terminal
 6. `nvm install 16` - installs NodeJS 16 and sets it as default
 7. `cd ~/git/flowcrypt-ios/appium && npm install`
+8. use Visual Studio Code IDE for editting appium tests - be sure to open it using `File` -> `Open Workspace from File` -> `Core/flowcrypt-mobile-core.code-workspace` (don't simply open the project as a folder, because advanced IDE functionality will be missing)
 
 ## Building app for testing
-
-Run this in `flowcrypt-ios` folder: `bundle exec fastlane build`. This will produce folder `appium/FlowCrypt.app` that contains the built app.
+1. Manually compile build from the current code:
+- run `bundle exec fastlane build` in `flowcrypt-ios` folder
+- it will produce `appium/FlowCrypt.app` for testing
+2. Use the latest simulator build:
+- copy `FlowCrypt.app` from `/DerivedData/FlowCrypt-.../Build/Products/Debug-iphonesimulator` (In Xcode open Products folder -> FlowCrypt -> Show in Finder).
```

Comment:
This looks like you need to take both step 1 and then step 2 in sequence. Which is not true. These two are two alternative approaches, and should be documented as such.